assets/src/scss/pages/post/_article-content.scss

```diff
@@ -219,6 +219,57 @@
   q {
     font-style: italic;
   }
+
+  .article-list-item {
+    .top-page-tags {
+      margin-bottom: $sp-1;
+    }
+
+    .article-list-item-headline {
+      padding-top: 0;
+      padding-bottom: 0;
+      -webkit-line-clamp: 2;
+    }
+
+    .article-list-item-content {
+      width: 100%;
+      -webkit-line-clamp: 3;
+
+      @include medium-and-up {
+        -webkit-line-clamp: 2;
+      }
+
+      @include large-and-up {
+        -webkit-line-clamp: 3;
+      }
+
+      @include x-large-and-up {
+        -webkit-line-clamp: 3;
+      }
+    }
+
+    .article-list-item-headline,
+    .article-list-item-content {
+      margin-bottom: $sp-1;
+      display: -webkit-box;
+      -webkit-box-orient: vertical;
+      overflow: hidden;
+    }
+
+    .article-list-item-image img {
+      @include medium-and-up {
+        height: 180px;
+      }
+
+      @include large-and-up {
```

Comment:
Unfortunately, we'll need to duplicate this rule from [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/blocks/Articles.scss#L44-L46), since the image's height is overridden. The other approach is to create a new `@media` only to M screens with something like this:

```
@mixin medium-only {
  @media (min-width: #{$medium-width}) and (max-width: #{$large-width - 1}) {
    @content;
  }
}
```
I'd suggest to implement it like this, and whenever we modify the `Articles block` (I hope soon) we would revert and standardize the code, by using only the [Articles styles](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/blocks/Articles.scss) and remove all these overrides.